### PR TITLE
[bitnami/mariadb-galera] Add podLabels as additional value option

### LINKED ISSUE #1873: [bitnami/mariadb-galera] Add support for setting pod labels
You can't set pod labels using values in the mariadb-galera chart. Currently it's difficult to use network policies with this chart as I can't set custom labels for multiple deployments.

### PR DESCRIPTION
**Pull request by connorearl**
**Description of the change**
Added an option to specify pod labels

**Benefits**
Allows for setting network policies by custom criteria, or anything else that uses labels for selection

**Possible drawbacks**
None

**Applicable issues**
  - fixes #1873 

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mariadb-galera/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb-galera
-version: 0.6.10
+version: 0.6.11
```
Comment (miguelaeh) [inline bitnami/mariadb-galera/Chart.yaml]: Please bump the minor version

File: bitnami/mariadb-galera/templates/_helpers.tpl [hunk 2]
```diff
@@ -35,6 +35,7 @@ app.kubernetes.io/name: {{ include "mariadb-galera.name" . }}
 helm.sh/chart: {{ include "mariadb-galera.chart" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{ toYaml .Values.podLabels }}
```
Comment (miguelaeh) [inline bitnami/mariadb-galera/templates/_helpers.tpl]:
```diff
- {{ toYaml .Values.podLabels }}
+ {{- toYaml .Values.podLabels -}}

**Comment by connorearl** [comment]
Thanks for the feedback, both changes made

File: bitnami/mariadb-galera/templates/_helpers.tpl [hunk 3]
```diff
@@ -35,6 +35,7 @@ app.kubernetes.io/name: {{ include "mariadb-galera.name" . }}
 helm.sh/chart: {{ include "mariadb-galera.chart" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- toYaml .Values.podLabels -}}
```
Comment (sameersbn) [inline bitnami/mariadb-galera/templates/_helpers.tpl]:
```suggestion
{{- if .Values.podLabels }}
{{- toYaml .Values.podLabels -}}
{{- end }}
```

**Pull request merged**